backend/code/archipelag/message/views.py

```diff
@@ -19,7 +21,26 @@ def message_create(request, market_id):
             form = MessageForm()
             service_name = event.type.service
             msg_for_user = "Twoja wiadomość na {} została dodana, dodaj wiadomości na inne platformy".format(service_name)
+            add_point_if_rules_allow(request.user.ngouser, market_id)
             return render(request, 'registration/message.html', {'form': form, 'message':msg_for_user})
     else:
         form = MessageForm()
-    return render(request, 'registration/message.html', {'form': form})
\ No newline at end of file
+    return render(request, 'registration/message.html', {'form': form})
+
+
+def add_point_if_rules_allow(ngo, market_id):
+    from archipelag.ngo.models import NgoUser
+    messages = Message.objects.filter(market_id=market_id).count()
+    if messages > 3:
+        coins_to_add = POINTS_RULES['create_more_than_three_messages_format']
+        ngo_model = NgoUser()
+        ngo_model.add_coins(ngo, coins_to_add)
+
+
+def add_point(request, market_id):
```

Comment:
this function is `add_point` but in body of this function i see only `add_coins`. It is an ambiguous

Reply:
changed to add coins